docs/SetupDatabaseForAKS.md

```diff
@@ -0,0 +1,75 @@
+# Set up a database for Scalar DB/Scalar DL deployment in AKS
+
+This guide explains how to set up a database for Scalar DB/Scalar DL deployment in AKS.
+
+## Cosmos DB
+
+In this section, you will create a Cosmos DB account.
+
+### Requirements
+
+* You must create a Cosmos DB account with `Core(SQL)` API.
+* You must change Cosmos DB `Default consistency` to `Strong`.
+* You must use Capacity mode as `Provisioned throughput`.
+
+### Recommendations
+
+* You should configure the backup policy as `Continuous` for Point-in-time-restore (PITR).
+* You should configure the service endpoint based on the [Azure official guide](https://docs.microsoft.com/en-us/azure/cosmos-db/how-to-configure-vnet-service-endpoint) to restrict access to Cosmos DB for production.
+
+### Steps
+
+* Create an Azure Cosmos DB account based on the [Azure official guide](https://docs.microsoft.com/en-us/azure/cosmos-db/create-cosmosdb-resources-portal#create-an-azure-cosmos-db-account) with the above requirements.
+
+### Scale Performance
+
+Cosmos DB performance can be adjusted using the following parameters.
+
+#### Request Units (RU)
```

Comment:
This and Autoscale sections explain kind of more than the other sections, which looks pretty weird as a doc since it's inconsistent. (Users would wonder why it explains more here and it doesn't really explain there.)

Reply:
@feeblefakie san
I have made some updates on the Request Units and Autoscalr section. We can't change the contents completely as it also discusses points related to scalar like specifying `cosmosBaseResourceUnit` and about scalar DL schema loader enables autoscale on created tables.